zerver/openapi/zulip.yaml

```diff
@@ -8420,6 +8420,29 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+    delete:
+      operationId: delete-custom-emoji
+      summary: Delete custom emoji
+      tags: ["server_and_organizations"]
+      description: |
+        This endpoint is used to delete a custom emoji from the user's
+        organization.
+
+        Note: Deleted emoji will still be visible in old messages.
+
+        **Changes**: New in Zulip 7.0 (feature level 162).
+      parameters:
+        - name: emoji_name
+          required: true
+          in: path
+          description: |
+            The name of the custom emoji to delete.
+          schema:
+            type: string
+          example: green_tick
+      responses:
+        "200":
+          $ref: "#/components/responses/SimpleSuccess"
```

Comment:
So the changes entry should be describing the API change -- which is just that we should document here a 404 response, and document as `**Changes**` that the error used to be a 400 status code.